include/ocpp/v201/evse.hpp

```diff
@@ -170,13 +168,13 @@ class Evse : public EvseInterface {
     Evse(const int32_t evse_id, const int32_t number_of_connectors, DeviceModel& device_model,
          std::shared_ptr<DatabaseHandler> database_handler,
          std::shared_ptr<ComponentStateManagerInterface> component_state_manager,
-         const std::function<void(const MeterValue& meter_value, const Transaction& transaction, const int32_t seq_no,
-                                  const std::optional<int32_t> reservation_id)>& transaction_meter_value_req,
-         const std::function<void()> pause_charging_callback);
+         const std::function<void(const MeterValue& meter_value, EnhancedTransaction& transaction)>&
```

Comment:
Fix the doxygen of the constructor so https://github.com/EVerest/libocpp/pull/539 can be closed